models/audience.js

```diff
@@ -20,7 +20,12 @@ const BigQueryMapping = {
 
 const JSONAttributes = ['properties']
 
-const ACSMapping = ['customer_email', 'geo_country']
+const ACSMapping ={
+  customer_email: ['Email for Connectors'],
+  geo_country: ['Geo Country Code']
```

Comment:
```Email for Connectors``` and ```Geo Country Code```are actually how the data comes in from Tealium. The mapping we did in Tealium doesn't seem to work.